lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -38,9 +38,19 @@ def descriptions(version)
     end
 
     def all_questions
-      dataset.keys.inject({}){|versions, version| versions.merge(questions(version))}
+      dataset.keys.inject({}) { |versions, version| versions.merge(questions(version)) }
     end
 
+    def is_london?(postcode)
+      area(postcode).any? { |result| result[:slug] == "london" }
+    end
+
+    def area(postcode)
+      response = Services.imminence_api.areas_for_postcode(postcode)
+      (response&.results || {})
+    end
```

Comment:
@floehopper We found that "results" returned from `imminence_api.areas_for_postcode` does not return a value for "code", however, [LandlordImmigrationCheckCalculator#areas_for_postcode](https://github.com/alphagov/smart-answers/blob/0f9d51b3022f10a071b07a393a9802e701293d21/lib/smart_answer/calculators/landlord_immigration_check_calculator.rb#L21-L24) checks for this value. We found this to be inconsistent and chose to implement it a different way.